### DISCUSSION TITLE
Initial load of test data on server start

### DISCUSSION THREAD

**Discussion by grdmnt** [shakacode/cypress-playwright-on-rails#129] [General]
Just trying to figure out if it's possible for cypress-on-rails to run some rails code to generate initial test data that will be used generally throughout the entire test suite? So instead of per test, I'm thinking of running a set of test data that will only run once at the start.

**Comment by Judahmeek** [comment]
@KhaledEmaraDev 
@grantspeelman 

Any clue?
